sql/pipelines/src/main/scala/org/apache/spark/sql/pipelines/graph/GraphValidations.scala

```diff
@@ -44,10 +44,13 @@ trait GraphValidations extends Logging {
       .foreach {
         case (dest, flows) =>
           throw new AnalysisException(
-            "MATERIALIZED_VIEW_WITH_MULTIPLE_QUERIES",
```

Comment:
I do think `MATERIALIZED_VIEW_WITH_MULTIPLE_QUERIES` would be a clearer error here, as the "materialized view"-ness is the real reason multiple queries aren't allowed (even though the code doesn't reflect this super well).